hash/nes.xml

```diff
@@ -56879,7 +56877,7 @@ preliminary proto for the PAL version, still running on NTSC systems) or the gfx
 	<software name="shenmoda" supported="no">
 		<description>Shen Mo Da Lu (Chi)</description>
```

Comment:
This should be "Shén Mó Dàlù"

Reply:
Shenmo is a set phrase. It is commonly not spaced. Example: https://sinolit.wordpress.com/2017/10/23/what-on-earth-are-shenmo-xiaoshuo